prac_07/guitar.py

```diff
@@ -1,6 +1,6 @@
 """
 CP1404 Practical
-Classes for guitars.py program
+Classes for guitars.py program.
 """
 
 
```

Comment:
You are missing the rest of your docstring for the __init__ for the Guitar class.
```
"""Initialise ..... """
```

I would recommend something like..

```
"""Initialize the Guitar object."""
```

Or something similar!